pkg/registry/relocator.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/name"
-	"github.com/google/go-containerregistry/pkg/v1"
```

Comment:
I feel like I am not processing some idiomatic Go stuff. would love to learn more (to avoid this)